packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneButton/DoneButton.js

```diff
@@ -1,37 +1,28 @@
-import { withThemeContext } from '@zooniverse/react-components'
+import { PrimaryButton } from '@zooniverse/react-components'
 import counterpart from 'counterpart'
-import { Button, Text } from 'grommet'
 import PropTypes from 'prop-types'
 import React from 'react'
 
 import en from './locales/en'
-import theme from './theme'
 
 counterpart.registerTranslations('en', en)
 
-const Label = (
-  <Text size='medium'>
-    {counterpart('DoneButton.done')}
-  </Text>
-)
-
 // TODO add back gold standard and demo buttons using grommet Button icon prop
 // {props.demoMode && <i className="fa fa-trash fa-fw" />}
 // {props.goldStandardMode && <i className="fa fa-star fa-fw" />}
 function DoneButton ({
-  annotations = [],
   completed = false,
   disabled = false,
-  hasNextStep = false,
   onClick = () => true
 }) {
   if (!completed) {
     return (
-      <Button
+      <PrimaryButton
+        color='green'
```

Comment:
Should this be "neutral-3" or "status-ok" (both are #1ED359)?

Reply:
The API for the `PrimaryButton` actually takes just simple color names. The prop `green` here will map to the correct theme color variable. We could consider changing this for consistency in the future maybe?

Reply:
Yep def not blocking, consider in the future sounds good to me.